recipes/seqan3/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "SeqAn3" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/seqan/{{ name|lower }}/releases/download/{{ version }}/seqan-3.0.0-with-submodules.tar.gz
+  sha256: ba8f0fd5d19289f95823235b69fdc5a12aba3578292463eecf750722e8662bd3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+
+test:
+  commands:
+    - test -e ${PREFIX}/include/seqan/align.h
+    - test -e ${PREFIX}/share/cmake/seqan/seqan-config.cmake
+    - test -e ${PREFIX}/share/pkgconfig/seqan-2.pc
+
+about:
+  home: http://docs.seqan.de/seqan/3-master-user/
```

Comment:
Our official website is https://www.seqan.de.

Can we provide a project url like https://github.com/seqan/seqan3 or a documentation link that would be http://docs.seqan.de/seqan/3-master-user/?

Reply:
I think whichever you'd prefer really, it's the link that comes up on the anaconda download page (e.g. https://anaconda.org/conda-forge/seqan-library), but command line install is the usual method so it's not usually seen

Reply:
>  Home: https://www.seqan.de
Development: https://github.com/seqan/seqan 

I think we can use the same format, just updating the Development url.